src/shared_state/crdts.md

```diff
@@ -0,0 +1,109 @@
+# CRDTs
+
+The previous section described a variety of errors that can occur when concurrent modifications are made to data that is stored in multiple locations.
+This section will present [**Conflict-free Replicated Data Types**](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type), a technology designed with the explicit goal of enabling exactly these types of behaviours.
+It will define the specialized terminology which seems to keep many from adopting CRDTs,
+provide some basic examples,
+lay out clear expectations of what they can and cannot accomplish,
+and hopefully prepare you to avoid concurrency-related bugs and enjoy the benefits of offline-first applications.
+
+## Terminology
+
+Just like in the conventional sense of a [Data Type](https://en.wikipedia.org/wiki/Data_type),
+a CRDT can be defined by the set of its possible values,
+the set of valid operations on those values,
+and low-level representation of those values.
+There are many possible types, and while some may be completely inapplicable to a given situation, there are often many ways to accomplish any particular goal.
```

Comment:
Here `types` refers to different sorts or variants of CRDTs right? If so, maybe pick another word since we are talking about data types and using _types_ could be somewhat confusing.

Reply:
Agreed, this wasn't very clear. I've rephrased the end of the paragraph entirely.